src/pyOpenMS/create_cpp_extension.py

```
@@ -221,9 +220,7 @@ def chunkIt(seq, num):
       for modname in mnames:
           fp.write("from .%s import *  # pylint: disable=wildcard-import; lgtm(py/polluting-import)\n" % modname)
 
-
-  # create version information
-  version = OPEN_MS_VERSION
-
-  print("version=%r\n" % version, file=open("pyopenms/_version.py", "w"))
-  print("info=%r\n" % QT_QMAKE_VERSION_INFO, file=open("pyopenms/_qt_version_info.py", "w"))
+  # create version and dependency information in a single file
+  with open("pyopenms/_dependency_version_info.py", "w") as f:
+          f.write(f"openms_version = {repr(OPEN_MS_VERSION)}\n")
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)